src/FlashMessage/FlashManager.php

```diff
@@ -68,29 +80,37 @@ public function handlesObject($object, ?string $statusName = null)
             return false;
         }
 
-        return \array_key_exists($statusName, $this->cssClasses);
+        return $this->handlesType($statusName);
     }
 
     /**
      * Returns the status CSS class for $object.
      *
+     * @deprecated since sonata-project/twig-extensions 1.x, will be removed in 2.0 version. Use getRenderedHtmlClassAttribute() instead.
+     *
+     * NEXT_MAJOR: remove this method
+     *
      * @param object|string $object     FlashManager or Sonata flash message type
      * @param string|null   $statusName Sonata flash message type
-     * @param string        $default    Default status class if flash message type do not exist
+     * @param string        $default    Default status class if Sonata flash message type do not exist
      *
      * @return string
      */
     public function getStatusClass($object, ?string $statusName = null, string $default = '')
     {
+        @trigger_error(sprintf(
+            'The "%s" methods is deprecated. Use "%s" instead.',
```

Comment:
```suggestion
            'The "%s()" method is deprecated since sonata-project/twig-extensions 1.x'
            .' and will be removed in version 2.0. Use "%s" instead.',
```